python/wipe_appwrite_bucket/README.md

```diff
@@ -0,0 +1,42 @@
+# 🚮 Wipe Appwrite Bucket
+
+This function deletes every file from a specified Appwrite Storage Bucket.
+
+## 🤖 Documentation
+
+_Example input:_
+
+```JSON
+{"bucketId":"profilePictures"}
+```
+
+_Example output:_
+
+```json
+{"success":true}
+```
+
+## 📝 Environment Variables
+
+List of environment variables used by this cloud function:
```

Comment:
Variables are no longer "Environment" variables so let's remove this for clarity:

```suggestion
## 📝 Variables

List of variables used by this cloud function:
```